app/assets/stylesheets/components/_select-with-search.scss

```diff
@@ -0,0 +1,416 @@
+.app-c-select-with-search {
+  //
+  // A theme for Choices.js styled to look like a GOV.UK Design System component
+  //
+  $background: #ffffff;
+  $focus-color: $govuk-focus-colour;
+  $focus-outline: $govuk-focus-width solid $govuk-focus-colour;
```

Comment:
I believe our standard is to keep all variables at the top.

Reply:
I didn't want to pollute the global namespace, which is why I kept them inside `.app-c-select-with-search { }`.

What do you recommend?

Reply:
Makes sense